### PR TITLE
Fixes #1848 - Link non-zero report summaries to the equivalent report search

### PR DESCRIPTION
**Pull request by GregSutcliffe**
I'd like to add some tests for this, but the current HostMailer tests don't load any reports, and I can't quite figure out how to add some....

### CODE REVIEW

File: app/views/host_mailer/_active_hosts.html.erb [hunk 1]
```diff
@@ -23,7 +23,11 @@
       <% else -%>
         <td style="border:1px solid #FF9933;border-collapse:collapse;padding:4px;background-color:#FFFFFF;">
       <% end -%>
-      <%= v %>
+      <% if v > 0 -%>
+        <%= link_to v, reports_path(:search=>"#{host} and #{m} > 0", :host => Setting[:foreman_url], :only_path => false) %>
```
Comment (GregSutcliffe) [inline app/views/host_mailer/_active_hosts.html.erb]: We don't seem to need " :host => Setting[:foreman_url] " on the link_to_host line further up in the file (https://github.com/GregSutcliffe/foreman/blob/639ad9ce4e4bb8da5e9aa0cfab8ac50f267b5b1b/app/views/host_mailer/_active_hosts.html.erb#L19). Why do I need to specify it here? If I just use ":host => host" in the same way, it creates the wrong links....

Comment (ohadlevy) [inline app/views/host_mailer/_active_hosts.html.erb]: which kind of wrong link?

Comment (GregSutcliffe) [inline app/views/host_mailer/_active_hosts.html.erb]: With the code as above, I get a URL like this for the skipped resources on "ircbot":

```
http://foreman/reports?search=ircbot.elysium.emeraldreverie.org+and+skipped+%3E+0
```

Which is correct. With ":host => host" I get:

```
http://ircbot.elysium.emeraldreverie.org/reports?search=ircbot.elysium.emeraldreverie.org+and+skipped+%3E+0
```

I.E the host URL is wrong. What I don't understand is that ":host => host" works for the first column:

```
https://foreman/hosts/ircbot.elysium.emeraldreverie.org
```

What am I missing?

**Comment by GregSutcliffe** [comment]
[test]

**Comment by GregSutcliffe** [comment]
Seems `@url` is sufficient, as it's set by HostMailer, so I've updated it.

**Comment by ohadlevy** [comment]
@GregSutcliffe is it still relevant for 1.3?

**Comment by GregSutcliffe** [comment]
Yes. I'll rebase on Monday

**Comment by GregSutcliffe** [comment]
@ohadlevy rebased

**Comment by ohadlevy** [comment]
merged as 3e7700b, thanks @GregSutcliffe

**Pull request closed without merging**